platform/darwin/src/MGLMultiPoint.mm

```diff
@@ -89,7 +89,7 @@ - (void)getCoordinates:(CLLocationCoordinate2D *)coords range:(NSRange)range
                                userInfo:nil] raise];
     }
 
-    assert(range.location + range.length <= _count);
+    NSAssert(range.location + range.length <= _count, @"Invalid coordinate range");
```

Comment:
Also describe the invalid range and indicate the correct count in the exception message, to aid in debugging.